automate github release with changelog

jstriebel opened the pull request:
### Description:
- Automates the github release with the changelog
- Also added some small improvements for the pypi metadata

### Issues:
- fixes #579

philippotto reviewed (approved):
Awesome! Only tested `changelog_for_version.sh`, but that works well for me.

I only noticed that the "clustertools" changelog will appear at the top (if there are changes). Since the focus of this repository is on wklibs + wkcuber, I'd rather change the order. Not sure how to encode this counter-lexicographically maybe? Or hardcode the order and do an additional `for PKG in */pyproject.toml where PKG not in hardcoded_pkgs` to pickup potentially existing other packages? No strong opinion on this, though.

jstriebel commented:
> Since the focus of this repository is on wklibs + wkcuber, I'd rather change the order. 

Good point, I just set a fixed order. Since I could directly use it, I added a Makefile with the most important make commands and moved all the various scripts in the `_tooling` subfolder. Could you give 37159b1 another look, @philippotto? Thanks a lot :pray:

philippotto reviewed (approved):
:tada:
